lib/npg_qc/autoqc/checks/rna_seqc.pm

```diff
@@ -211,40 +211,33 @@ sub _command {
     return $command;
 }
 
-override 'can_run' => sub {
+override 'execute' => sub {
     my $self = shift;
-    if (! $self->_annotation_gtf) {
-        $self->result->add_comment(q[No GTF annotation available]);
-        return 0;
-    }
-    return 1;
-};
 
-override 'execute' => sub {
-    my ($self) = @_;
-    my @comments;
-    my $can_execute = 1;
     if (super() == 0) {
-    	return 1;
+        return 1;
     }
     $self->result->set_info('Jar', qq[RNA-SeqQC $RNASEQC_JAR_NAME]);
+
+    my @comments = ();
     if (! $self->_ref_genome) {
         push @comments, q[No reference genome available];
-        $can_execute = 0;
     }
     if(! $self->_alignments_in_bam) {
         push @comments, q[BAM file is not aligned];
-        $can_execute = 0;
     }
     if (! $self->_is_rna_alignment) {
         push @comments, q[BAM file is not RNA alignment];
-        $can_execute = 0;
     }
-    if (! $can_execute || ! $self->can_run()) {
-        my $can_run_message = join q[; ], @comments;
-        $self->result->add_comment($can_run_message);
+    if (@comments) {
+        $self->result->add_comment(join q[; ], @comments);
         return 1;
     }
+
+    if (! $self->_annotation_gtf) {
```

Comment:
small but growing number of species supported - given low [RNA alignments numbers](https://github.com/wtsi-npg/npg_seq_pipeline/pull/206) currently might be better to leave this in can_run ?